Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ target_link_libraries(mesh_simplification_plugin PUBLIC scene_surface_mesh_item
 
 qt5_wrap_ui( remeshingUI_FILES  Remeshing_dialog.ui)
 polyhedron_demo_plugin(offset_meshing_plugin Offset_meshing_plugin ${remeshingUI_FILES})
-target_link_libraries(offset_meshing_plugin PUBLIC scene_surface_mesh_item scene_polygon_soup_item)
+target_link_libraries(offset_meshing_plugin PUBLIC polyhedron_demo scene_surface_mesh_item scene_polygon_soup_item)
```

Comment:
Once `lock_test_item` is moved to an interface, that link to `polyhedron_demo` will not be necessary.